src/App.tsx

```diff
@@ -1,8 +1,11 @@
+/**@jsx jsx */
+import { Global, jsx } from '@emotion/react'
```

Comment:
Why do we need to use their `jsx`? we agreed in #24 to use styled syntax

do we still need to use it with the styled syntax?

Reply:
i checked package.json and both emotion/react and emotion/styled were installed. so I think we will be using both.

Reply:
> 
> 
> Why do we need to use their `jsx`? we agreed in #24 to use styled syntax
> 
> do we still need to use it with the styled syntax?

because globleStyled = css`....` and it doesnt work if @jsx is not included

Reply:
we need `@emotion/react` for its `Global` and `css` exports but we agreed to only use the styled syntax for styling

Reply:
I don't think theres a way to do the Global without the css prop https://github.com/emotion-js/emotion/issues/1262

Reply:
@merodiro ow okay =) thanks for clearing that out. @YPAzevedo Thanks ^_^